ansible/roles/jupyter-repo2docker/tasks/repo2docker.yml

```diff
@@ -12,8 +12,13 @@
 
 - name: Install repo2docker
   pip:
-    name: 
+    name:
       - jupyter-repo2docker
+    version: "2022.10.0"
```

Comment:
This is a worry, I wonder if we are just missing a dependency with the latest version? Can we did into this one a bit more please? I think having the older base image is probably better than an older repo2docker, from a security point of view.

Reply:
Resolved by pinning repo2podman to git.

Reply:
Can you pin to a specific commit SHA in lieu of a recent tag?

Reply:
Done